contracts/core/exchange-wrappers/KyberNetworkWrapper.sol

```diff
@@ -139,33 +141,27 @@ contract KyberNetworkWrapper {
 
         // Ensure the issuance order maker token is allowed to be transferred by KyberNetworkProxy as the source token
         ERC20.ensureAllowance(
-            _makerToken,
+            _addresses[2],
```

Comment:
For clarity, I would instantiating these array addresses/uints as variables before usage. As I read the code, it is jarring to go back and forth to the comments above to understand what is going on.